src/backtrace/libunwind.rs

```diff
@@ -51,6 +51,16 @@ impl Frame {
         unsafe { uw::_Unwind_GetIP(ctx) as *mut c_void }
     }
 
+    pub fn gr(&self, index: u32) -> Option<usize> {
+        let ctx = match *self {
+            Frame::Raw(ctx) => ctx,
+            Frame::Cloned { .. } => {
+                return None;
```

Comment:
This seems like it may be a bit of a hazard at  some point, but I think it's fine for now.